src/BasicVertexShader.glsl

```diff
@@ -1,12 +1,14 @@
 #version 330 core
 
-layout(location = 0) in vec3 vertexPosition_modelspace;
-layout(location = 1) in vec4 vertexColour;
+layout(location = 0) uniform mat3 projectionView;
```

Comment:
I thought this was going to be a single `mat4`?

Reply:
It is. This isn't updated yet.